📰 Write a blog post calling for volunteers for 2023 (#235)
djangocon/2022.djangocon.us: _posts/2022-10-18-call-for-2023-volunteers.md

---
author: Drew Winstel
category: General

> date: 2022-10-18 11:00:00-07:00

Reviewer: I would make sure this date is newer than whatever time you publish it or it may not show up until the next time we publish something.

Author: Yep, planning on it.

- Work with social events team to arrange speaker appreciation dinner
- Generate spreadsheets in multiple formats to make other teams' lives easier

### At-conference responsibilities

- Work with registration desk to make sure that all speakers show up
- Work with session managers (other organizers) to ensure distribution of cue sheets and speaker gifts
- Check with tutorial speakers to make sure they have everything they need

## Opportunity Grants Committee

Size: 2 (plus reviewers)

The OG committee is responsible for obtaining grants from the DSF and PSF and awarding grants to people who will best use them, prioritizing speakers, first time attendees, and people from underrepresented and/or historically disadvantaged groups.

### Pre-conference responsibilities

- Start call for grants process (concurrent with CFP)
- Publish blog posts announcing grants process
- Request grants from Django Software Foundation and Python Software Foundation (the DEFNA board will help you fill out the forms)
- Recruit grant reviewers (other organizers and DEFNA members are happy to help)
- Review submissions

> - Coordinate with program team to award to speakers

Reviewer 2: ```suggestion
- Coordinate with program team about grants to speakers
```

- Notify grant recipients in waves (concurrent with program notifications)
- Communicate with recipients on how to get reimbursed (we'll have templates)
- Give DEFNA a lit of recipients and amounts
- Approve reimbursements as requests come in
- Coordinate with visa (community) team as needed to approve invitation letter requests
- Write transparency report giving a high-level overview of how the grants were awarded

### At-conference responsibiities

- Answer questions from grant recipients and the DEFNA board

## Community Team

Size: 2-4

### Pre-conference responsibilities

- Schedule orientation
- Arrange childcare and lactation room
- Coordinate visa applications
- Improve accessibility of the venue and conference as a whole

### At-confrence responsibilities

- Post sign-ups for lactation room and distribute keys as needed
- Collect receipts for childcare reimbursement

## Sponsorship Committtee

Size: 2-4

The sponsorship committee recruits sponsors which help offset the cost of running DCUS.

### Pre-conference responsibilities

> - Create and announce sponsorship prospectus

Reviewer 2: ```suggestion
- Create and announce sponsorship prospectus (we have templates, and DEFNA will help with specifics)
```

- Recruit sponsors
- Coordinate contract signing with DEFNA
- Consider awarding community sponsorships (non-financial) to other non-profit Python conferences and related communities (local Python meetups near the conference will love the exposure)
- Coordinate shipping of sponsor swag for the swag bag
- Lay out sponsor tables (on a venue map)
- Work with sponsor chair/co-chair to handle sponsor shout-outs if needed
- Order banners to go behind the speakers on stage

### At-conference responsibilities

- Direct sponsors to their booths/tables and swag

> - Remind non-sponsors (or sponsors who haven't procured a booth) that they can't have a booth if needed

Reviewer 2: ```suggestion
- Ensure that booths are reserved for sponsors who have reserved a booth, and are not open for others
```

- Field complaints/questions from sponsors

## Code of Conduct Committee

Size: 4-6

The CoC committee suggests improvements to the CoC and investigates reports. This is the heart of the conference because without the CoC, we wouldn't have the welcoming environment that we have at DCUS.

### Pre-conference responsibilities

- Suggest modifications/improvements to the CoC
- Assist program committee with pre-conference review of recorded online talks

### At-conference responsibilities

> - Field reports and investigate them, including recommending actions to the DEFNA board

Reviewer 2: ```suggestion
- Field reports and investigate them, including recommending actions to the DEFNA board
- Follow-up on reports, including conversations with both reporters and those who have been reported, and facilitate the appropriate response
```

- Plan opening reception (if sponsored, this will become a bigger event compared to previous years)
- Plan any other fun events like board game night

### At-conferece responsibilities

- Work with venue on any work needed for social events

## Venue Committee

Size: 1-2

The venue committee is responsible for coordinating plans between the other organizing teams and the venue(s).

### Pre-conference responsibilities

- Work with event venue to select and lay out talk, tutorial, and sprint rooms
- Lay out the extra rooms needed for the conference (lactation room, quiet room, organizers room, speaker green room, lunch/break areas)
- Work with event venue and DEFNA to select catering menus (and providers if we're not contractually obligated to use one)
- Work with hotel to coordinate room blocks
- Work with venue to plan logistics (sponsor gear shipping, signage, etc.)
- Allocate reward rooms in coordination with DEFNA board
- Order supplies for restroom stocking

### At-conference responsibilities

- Review room guest list against ticket list to claim reward nights
- Handle any last-minute requests from vendors/sponsors/organizers/etc.

> - Notify venue staff of any needs (bathroom cleanliness, etc.)

Reviewer 2: ```suggestion
- Notify venue staff of any needs (bathroom cleanliness, meals to accommodate special diets, etc.)
```
